doc/source/data/doc_code/creating_datasets.py

```diff
@@ -593,6 +593,16 @@
 # __read_parquet_az_end__
 # fmt: on
 
+# fmt: off
+# __read_compressed_begin__
+# Read a gzip compressed Parquet file from S3.
+ds = ray.data.read_parquet(
+  "s3://path/my_large_parquet_file.gz",
```

Comment:
Could you upload a file to `air-example-data` or some other public bucket so that this can be tested in CI?

See also: https://github.com/ray-project/ray/pull/33543

Reply:
Could you add `anonymous` to the URI like  `s3://anonymous@air-example-data/my_large_parquet_file.gz`? For some reason, CI fails if you exclude it.

Other than that, could you just check that it works locally? If there are still issues, I'll address them in https://github.com/ray-project/ray/pull/33543 (the code isn't currently tested in CI)

Reply:
confirmed that it works locally, let me know if you run into issues down the line!